cmd/auth-provider-gcp/main.go

```diff
@@ -0,0 +1,9 @@
+package main
```

Comment:
This will need a CNCF copyright/license block at the beginning.

Reply:
Done